docs/user/sbt.rst

```diff
@@ -39,8 +39,6 @@ Sbt settings
 ======================== =============== =======================================
 Name                     Type            Description
 ======================== =============== =======================================
-``nativeClang``          ``File``        Path to ``clang`` command
-``nativeClangPP``        ``File``        Path to ``clang++`` command
 ``nativeCompileOptions`` ``Seq[String]`` Extra options passed to clang verbatim
```

Comment:
That's unfortunate, but having it as a setting locks us out of doing anything non-trivial to generate the default value.